tests/slr_integration/Orders/__snapshots__/Controller_Test__test_attendee_list_seat_column__single event with 3 regular Ticket attendees__0.snapshot.html

```diff
@@ -109,7 +109,7 @@ <h3>Attendance Overview</h3>
 			<div class="tec-admin__upsell-icon">
 	<img
 		class="tec-admin__upsell-icon-image"
-		src="http://wordpress.test/wp-content/plugins/the-events-calendar/common/src/resources/images/icons/circle-bolt.svg"
+		src="http://wordpress.test/wp-content/plugins/event-tickets/common/src/resources/images/icons/circle-bolt.svg"
```

Comment:
@lucatume mostly a question - is that `event-tickets` or `the-events-calendar` unreliable ? It could change based on which common is at the latest version ?

Reply:
You are correct, the Common URLs will change depending from where it's loaded.
One way to "fix" the snapshots is to [ensure Common URLs will look like coming from the plugin under test](https://github.com/the-events-calendar/event-tickets/pull/3140/commits/8220b5692ca422a32450893606cfec8b60251462).